api/api/src/main/scala/uk/ac/wellcome/platform/api/services/ElastsearchSearchRequestBuilder.scala

```diff
@@ -34,49 +34,56 @@ case class ElastsearchSearchRequestBuilder(
 
   lazy val aggregations = queryOptions.aggregations.map {
     case AggregationRequest.WorkType =>
-      CompositeAggregation("workType").sources(
-        List(
-          TermsValueSource("label", field = Some("workType.label.raw")),
-          TermsValueSource("id", field = Some("workType.id")),
-          TermsValueSource("type", field = Some("workType.ontologyType"))
+      CompositeAggregation("workType")
+        .size(Some(100))
+        .sources(
+          List(
+            TermsValueSource("label", field = Some("workType.label.raw")),
+            TermsValueSource("id", field = Some("workType.id")),
+            TermsValueSource("type", field = Some("workType.ontologyType"))
+          )
         )
-      )
+
     case AggregationRequest.ProductionDate =>
-      // We use `productionDates` here over `production.dates` to match the case classes, which we then serialise to
-      // the JSON path later.
```

Comment:
:point_up: removed this comment as this class is just for building a request for elasticsearch, and beyond the scope of this class to know about how serialisation is performed further down the line.